src/proteus/dswx_hls.py

```diff
@@ -1939,6 +1958,68 @@ def save_dswx_product(wtr, output_file, dswx_metadata_dict, geotransform,
     logger.info(f'file saved: {output_file}')
 
 
+def geotiff2png(src_geotiff_filename,
+                dest_png_filename,
+                output_height=-1,
+                output_width=-1,
```

Comment:
If you want to make these arguments optional, you probably need to handle them in case they are not provided by the user. In the rest of the code, you used the default value `None`. Is there a reason, why you are using `-1` here?

Reply:
Just noticed that you fixed it in the last commit.

Reply:
Oof, yeah. My apologies for that. I just went back to look at something, and realized I never handled those cases, and that `scratch_dir` never got used. (sigh) But really good catch! Thank you!